config/packages/doctrine.yaml

```diff
@@ -22,6 +22,8 @@ doctrine:
         validate_xml_mapping: true
         naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
         auto_mapping: true
+        controller_resolver:
+            auto_mapping: true
```

Comment:
Set to `false` and use MapEntity: https://symfony.com/doc/current/doctrine.html#mapentity-options. Which is the purpose from Symfony to deprecate this auto mapping.

Reply:
yes that's a better way to go at dealing with this deprecation and it should be done, but that would need a bit of more thorough check over routes, controllers and such, and that'd definitely take some time, after that's taken care of then this could be changed to `false` like suggested

so the change here is just to make sure that things just doesn't suddenly break and went up in flames when the defaults finally changed